ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -795,7 +795,7 @@ aaa accounting commands 0 console start-stop logging
 aaa accounting commands 1 console start-stop group TACACS1
 aaa accounting exec default start-stop group TACACS logging
 aaa accounting system default start-stop group TACACS
-aaa accounting dot1x default start-stop group RADIUS
+aaa accounting dot1x default stop-only logging group group1 group group2 multicast logging
```

Comment:
I dont think it will work via CVP since the mentiond groups are not exist in cfg file. Please either use the available group or add few more groups in aaa_server_groups host.

Reply:
Added group from aaa_server_group.yml file.

Reply:
The group names are not correct. It should be RADIUS1/2 not Radius1/2. Plus do the same for host2.cfg

Reply:
Fixed.